src/BlockTypes/ProductCategories.php

```diff
@@ -77,9 +86,11 @@ protected function render( $attributes, $content ) {
 			}
 		}
 
-		$classes = $this->get_container_classes( $attributes );
+		$classes_and_styles = $this->get_container_classes( $attributes );
+		$classes            = $classes_and_styles[0];
+		$styles             = $classes_and_styles[1];
 
-		$output  = '<div class="' . esc_attr( $classes ) . '">';
+		$output  = '<div class="wp-block-woocommerce-product-categories ' . esc_attr( $classes ) . '" style="' . esc_attr( $styles ) . '">';
```

Comment:
There is a possibility that the style property will be empty. Do you think that is it necessary to exclude style property when the variable $styles is empty?
The same concept for class and variable$classes

Reply:
@gigitux  Empty style or class attribute is fine to me. But if you prefer a cleaner HTML output, go for it. I prefer empty attributes because of the readability. We will need some `if` condition checks to exclude the empty attributes, which makes the code harder to read IMO.

Reply:
>We will need some if condition checks to exclude the empty attributes, which makes the code harder to read IMO.

I agree with you! In any case, I preferred to point it out to you